### PR TITLE
Add jfrog icons for xray and artifactory

### PR DESCRIPTION
**Pull request by jsamuel1**
Added jfrog icons for JFrog, JFrog XRay and JFrog Artifactory.
Sourced from the media kit linked to from https://jfrog.com/brand-guidelines/
Resized to 64x64 using imagemagick command:
"mogrify -resize 64x64 -background transparent -gravity Center -extent 64x64 -format png -strip "...

Source files are:
* Icon_AF_Green_RGB.png
* Icon_Xray_Green_RGB.png
* Jfrog_Green_RGB.png

Did not create an alias from jfrog-xray to xray, due to known name conflicts.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +27,10 @@ Buildkite emoji will be shown on both light or dark backgrounds, and at a small
 
 Emoji | Aliases
 ----- | -------
+<img src="img-buildkite-64/jfrog-conan.png" width="20" height="20" alt="jfrog-conan"/> | `:jfrog-conan: :conan: :conanio:`
```
Comment (ticky) [inline README.md]:
```suggestion
<img src="img-buildkite-64/jfrog-conan.png" width="20" height="20" alt="jfrog-conan"/> | `:jfrog-conan:`, `:conan:`, `:conanio:`
```

File: README.md [hunk 2]
```diff
@@ -27,6 +27,10 @@ Buildkite emoji will be shown on both light or dark backgrounds, and at a small
...
+<img src="img-buildkite-64/jfrog.png" width="20" height="20" alt="jfrog"/> | `:jfrog:`
+<img src="img-buildkite-64/jfrog-artifactory.png" width="20" height="20" alt="jfrog-artifactory"/> | `:jfrog-artifactory: :artifactory:`
```
Comment (ticky) [inline README.md]:
```suggestion
<img src="img-buildkite-64/jfrog-artifactory.png" width="20" height="20" alt="jfrog-artifactory"/> | `:jfrog-artifactory:`, `:artifactory:`
```

File: img-buildkite-64.json [hunk 3]
```diff
@@ -1,4 +1,33 @@
 [
+  {
+    "name": "jfrog-conan",
+    "image": "img-buildkite-64/jfrog-conan.png",
+    "category": "Buildkite",
+    "aliases": [
+    "conan",
```
Comment (ticky) [inline img-buildkite-64.json]:
```suggestion
      "conan",
```

File: img-buildkite-64.json [hunk 4]
```diff
@@ -1,4 +1,33 @@
...
+    "conanio"
```
Comment (ticky) [inline img-buildkite-64.json]:
```suggestion
      "conanio"
```

**Comment by jsamuel1** [comment]
> A few housekeeping changes;
> 
> * The lists of shortcodes are all joined, I've added suggested edits to make them each a separate code element
> * The indentation on the aliases for `:jfrog-conan:` was off, and would generate unnecessary diffs later on if kept this way, suggested edits fix this
> 
> Otherwise looks good!

Thanks @ticky.  Suggestions commited, and merge conflicts with my other PR now resolved.

**Comment by ticky** [comment]
Thanks so much! I’ll get these released shortly!

**Pull request merged**